src/utils/Card/index.js

```diff
@@ -1,12 +1,13 @@
 import React, { PropTypes } from 'react'
 import { keys } from 'lodash'
 import styled from 'styled-components'
-import { CourseMeta, CourseHeader } from './CourseCard'
-import { LessonMeta, LessonHeader } from './LessonCard'
-import { PlaylistMeta, PlaylistHeader } from './PlaylistCard'
+import { CourseMeta, CourseHeader } from '../../components/CourseCard'
+import { LessonMeta, LessonHeader } from '../../components/LessonCard'
+import { PlaylistMeta, PlaylistHeader } from '../../components/PlaylistCard'
```

Comment:
I would recommend the `Card` and `*Card` components be refactored; IMO, Card shouldn't have to import all of these pieces unless it is exporting or using them. I would recommend a single `Card` component that exports the different types so you can do `import {PlaylistCard} from 'Card'` and keep everything clean without side effects like this.